File: mega-zep-frontend/src/main/angular/frontend/src/app/modules/shared/models/NewCommentEntry.ts
import {Employee} from './Employee';

export class NewCommentEntry {
  stepId: number;
  employee: Employee;
  comment: string;
  assigneeEmail: string;
  project: string;
  currentMonthYear: string;

  constructor(stepId: number, employee: Employee, comment: string, assigneEmail: string, project: string, currentMonthYear: string) {
    this.stepId = stepId;
    this.employee = employee;
    this.comment = comment;
    this.assigneeEmail = assigneEmail;
    this.project = project;
    this.currentMonthYear = currentMonthYear;
  }
}

File: mega-zep-backend/src/main/java/com/gepardec/mega/rest/model/NewCommentEntry.java
package com.gepardec.mega.rest.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.gepardec.mega.domain.model.Employee;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.ToString;
import lombok.experimental.Accessors;
import lombok.extern.jackson.Jacksonized;

@Builder(builderClassName = "Builder")
@Getter
@ToString
@EqualsAndHashCode
@Accessors(fluent = true)
@JsonIgnoreProperties(ignoreUnknown = true)
@Jacksonized
public class NewCommentEntry {

    @JsonProperty
    private final Long stepId;

    @JsonProperty
    private final Employee employee;

    @JsonProperty
    private final String comment;

    @JsonProperty
    private final String assigneeEmail;

    @JsonProperty
    private final String project;

    @JsonProperty
    private final String currentMonthYear;
}
